dev-client/src/screens/SoilScreen/SoilScreen.tsx

```diff
@@ -92,23 +70,58 @@ export const SoilScreen = ({siteId}: {siteId: string}) => {
     [siteId, dispatch],
   );
 
+  const updateSoilDataDepthPreset = useCallback(
+    (newDepthPreset: SoilIdSoilDataDepthIntervalPresetChoices) => {
+      dispatch(updateSoilData({siteId, depthIntervalPreset: newDepthPreset}));
+    },
+    [dispatch, siteId],
+  );
+
   return (
-    <BottomSheetModalProvider>
-      <ScrollView backgroundColor="grey.300">
-        <Row backgroundColor="background.default" px="16px" py="12px">
-          <Heading variant="h6">{t('soil.surface')}</Heading>
-        </Row>
-        <Box height="16px" />
-        <Row backgroundColor="background.default" px="16px" py="12px">
-          <Heading variant="h6">{t('soil.pit')}</Heading>
-        </Row>
-        {allIntervals.map(interval => (
-          <DepthIntervalView
-            key={`${interval.depthInterval.start}:${interval.depthInterval.end}`}
-            siteId={siteId}
-            depthInterval={interval}
-          />
-        ))}
+    <ScrollView backgroundColor="grey.300">
+      <SoilSurfaceStatus
+        required={project ? project.verticalCrackingRequired : true}
+        complete={Boolean(soilData?.surfaceCracksSelect)}
+        siteId={siteId}
+      />
+      <Box height="16px" />
+      <Row
+        backgroundColor="background.default"
+        px="16px"
+        py="12px"
+        justifyContent="space-between">
+        <Heading variant="h6">{t('soil.pit')}</Heading>
+        {soilData && !project && (
+          <BottomSheetModal
+            trigger={onOpen => (
+              <IconButton
+                name="tune"
+                _icon={{color: 'action.active'}}
+                onPress={onOpen}
+              />
+            )}>
+            <EditSiteSoilDepthPreset
+              /* LANDPKS is the default here */
+              selected={soilData.depthIntervalPreset || 'LANDPKS'}
```

Comment:
issue: use a constant instead of a bare string. also: if there are other places where LANDPKS is the default preset, reference the same constant.

Reply:
see [previous comment](https://github.com/techmatters/terraso-mobile-client/pull/758#discussion_r1456512597)